### DISCUSSION TITLE
Launch the local session

### DISCUSSION THREAD

**Discussion by XiongHuyu** [ansys/pyansys-geometry#1285] [Q&A] [answered]
Hello,
I'm trying to run SCDM with the script below(The Python version is 3.11.7, and the SCDM version is 23R2.)，but SCDM is not responding after startup.
```
from ansys.geometry.core.math import Plane, Point3D, Point2D
from ansys.geometry.core.misc import UNITS, Distance
from ansys.geometry.core.sketch import Sketch

# Define a sketch
origin = Point3D([0, 0, 10])
plane = Plane(origin, direction_x=[1, 0, 0], direction_y=[0, 1, 0])

# Create a sketch
sketch = Sketch(plane)
sketch.circle(Point2D([1, 1]), 30 * UNITS.m)
sketch.plot()

# Start a modeler session
from ansys.geometry.core import launch_modeler_with_spaceclaim
from ansys.geometry.core import Modeler
modeler = launch_modeler_with_spaceclaim()
m = Modeler()
# Create a design
design = m.create_design(name="Design1")

# Create a body directly on the design by extruding the sketch
body = design.extrude_sketch(
    name="CylinderBody", sketch=sketch, distance=Distance(80, unit=UNITS.m)
)

# Plot the body
design.plot()

# Export the model to SCDOCX format
file_path = design.export_to_scdocx()
```
Is there any solution?
Thanks!

**Comment by RobPasMue** [comment]
Hi @XiongHuyu! Here is the Ansys Version compatibility table.. https://geometry.docs.pyansys.com/version/0.6/getting_started/compatibility.html#ansys-version-compatibility

Support for 23R2 is very limited - only versions 0.3.X of PyAnsys Geometry support it and it is very prompt to failure. The first fully supported version is 24R1, and thus versions 0.4.X and onwards of PyAnsys Geometry.

Sorry for the inconveniences!

**Reply by XiongHuyu** [reply]
Thank you for your answer @RobPasMue ！I tried version 0.3.X of PyAnsysGeometry, but it didn't work, so the only solution is to install version 24R1 of SCDM？

**Reply by RobPasMue** [reply]
Hi @XiongHuyu! Yes, I'd recommend you upgrade to version 24R1 and then you could use the latest version of PyAnsys Geometry. Support with 23R2 is very limited and as you just experienced, it is prompt to crashing.

**Reply by XiongHuyu** [reply]
Hi @RobPasMue!I have updated to version 24R1, but there is still the problem, after starting the local SCDM, it does not seem to connect. I run the following script.
```
from ansys.geometry.core import launch_modeler_with_spaceclaim
medeler = launch_modeler_with_spaceclaim()
print(medeler)
```
The error is as follows:
_ConnectionError: Timeout while waiting for backend to be ready._

**Reply by RobPasMue** [reply]
Hi @XiongHuyu! Couple of questions:

* Does SpaceClaim at least start?
* Which version of PyAnsys Geometry are you using now? (``pip freeze`` and search for ``ansys-geometry-core``)

**Reply by XiongHuyu** [reply]
Hi @RobPasMue! Yes, it started and then closed after a while, indicating that the connection timed out. I pip installed PyAnsys Geometry version 0.6.4.
![BFA2BC07-4E3E-4740-A2E5-8AD6C5845BB5](https://github.com/ansys/pyansys-geometry/assets/174261867/3ad4882b-3891-4e9a-a04a-4074d1f29e46)

**Reply by RobPasMue** [reply]
Interesting, let me try to replicate it on my side. Quick question as well for you. Do you have write access to you C:\ folder?

**Reply by XiongHuyu** [reply]
Yes, I just lifted the C:\ folder permissions restriction and it still does the above.

**Reply by RobPasMue** [reply]
There might be a bug... looking into it =)

**Reply by RobPasMue** [reply]
@XiongHuyu there is a bug in the code introduced in versions 0.6 and onwards. I am currently fixing it and will do a patch release asap (0.6.5). Please stay tuned. See #1289

**Reply by RobPasMue** [reply, marked as answer]
Hi @XiongHuyu! Version 0.6.5 has just been released and it will fix your issue. Please give it a try! This problem affected version 0.6.4 only.

**Reply by XiongHuyu** [reply]
Hi @RobPasMue!It's so cool! I managed to start SCDM to build a extrude model. Thanks for your help! Also, does the 24R1 version of SCDM not support PyAnsys Geometry using rotation operations?

**Reply by RobPasMue** [reply]
Hi @XiongHuyu! I'm glad you made it work =)

Unfortunately, rotate operations are only available starting on 24R2 😢 but hey... good news is that 24R2 is just a few days away!! 😄

**Reply by XiongHuyu** [reply]
Well, I look forward to the release of the 24R2, and thanks again for your help.
